.circleci/config.yml

```diff
@@ -199,17 +213,36 @@ jobs:
         description: Docker repo
         type: string
         default: "oplabs-tools-artifacts/images"
-    docker:
-      - image: cimg/python:3.7
-    resource_class: small
+    machine:
+      image: ubuntu-2204:2022.07.1
+      resource_class: xlarge
     steps:
       - gcp-cli/install
       - gcp-oidc-authenticate
       - checkout
       - run:
-          name: Tag
+          name: Build
```

Comment:
I'm fine with this being inlined but also may be better to have more complex logic in a script and pass in args

Reply:
I can clean this up in separate PRs - need to see it run against develop first.